Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -31,9 +32,21 @@ class Digitizer
   void setSrcID(int sourceID) { mSrcID = sourceID; }
 
  private:
+  TRDGeometry* mGeom = nullptr;
+
   double mTime = 0.;
   int mEventID = 0;
   int mSrcID = 0;
+
+  float mWion; //  Ionization potential
+  // int hitLoopBegin; // Helper variable for sorting hits
+
+  bool SortHits(std::vector<o2::trd::HitType>&);
+  bool GetHitContainer(const int, const std::vector<o2::trd::HitType>&,
+                       std::vector<o2::trd::HitType>&);
+  bool CheckHitContainer(const int, const std::vector<o2::trd::HitType>&);
+  bool ConvertHits(int det, const std::vector<o2::trd::HitType>&, int&);
+  bool Diffusion(float, double, double, double&, double&, double&);
```

Comment:
function names should start with a lower case (action verb). I know that this is not consistent in O2 but new code should follow the coding guidelines.

Reply:
could you document what the 'bool' return type means in each case?

Reply:
That's an ugly convention but I will follow it.